doc/admin-guide/files/records.config.en.rst

```diff
@@ -4407,6 +4407,31 @@ Sockets
    platforms.  (Currently only linux).  IO buffers are allocated with the MADV_DONTDUMP
    with madvise() on linux platforms that support MADV_DONTDUMP.  Enabled by default.
 
+.. ts:cv:: CONFIG proxy.config.ssl.misc.io.max_buffer_index INT 8
+
+   Configures the max IOBuffer Block index used for various SSL Operations
+   such as Handshake or Protocol Probe. Default value is 8 which maps to a 32K buffer
```

Comment:
Do we have a table for indexes and sizes on the docs? There should be `define` or `enum` in the code, but it's not very user friendly.

Reply:
Yeah, good point, but, could I do it in a separate PR later? Need to figure out the right doc to put that in and also the formatting etc.

Reply:
Sure, I don't think it's a blocker.